patterns/1-initial/principles.md

```diff
@@ -0,0 +1,152 @@
+though with people lacking an open source background. As a remedy the most
+important principles of InnerSource get published widely.
+
+## Problem
+
+The organisation is trying to roll out InnerSource at a larger scale. The
+initiative started among open source enthusiasts. The goal is now to get buy-in
+from people that are lacking open source experience. For that audience the typical
+slogan of "applying open source best practices" is no longer sufficient to
+transport the message of what InnerSource is, which problems it solves and which
+tools it uses for solving these issues. As a result InnerSource adoption in the
+organisation slows down. Teams develop diverging ideas of what the goals of InnerSource
+is about and how to best implement it leading to confusion when contributors
+are starting to cross team boundaries. 
+
+## Story
+
+Early experiments in an organisation have shown that open source collaboration
+best practices can be beneficial. The goal now is to move the initiative to
+teams and individuals lacking a deep background in open source.
+
+The goal now is to clearly communicate the goals of the InnerSource initiative
+as well as a clear path towards achieving these goals.
+
+## Forces
+
+* Teams have trouble communicating exactly what the important aspects of
+InnerSource are.
+
+* People lacking open source experience fail to understand what it means to bring
+open source best practices into the organisation.
+
+* On a daily basis teams trying to follow InnerSource best practices have a hard
+time deciding if what they are doing is inline with general InnerSource values.
+
+## Solution
+
+Those that should adopt InnerSource need to understand two perspectives:
+
+### Why does the organisation want to adopt InnerSource?
+
+In the past InnerSource has proven to be successful to solve several issues
+commonly found in organisations. Instead of going for generalizations, try to
+exactly identify the solutions that match the challenges of your organisation -
+preferably with those affected by the change you want to see. 
+
+Some challenges that others have addressed by following InnerSource 
+best practices:
+
+* Reduce development silos caused by excessive ownership culture.
+* Increase innovation speed by reducing time spent solving similar issues by
+  fostering healthy code reuse.
+* Increase development speed by better cross team collaboration.
+* Solve project/ team dependencies beyond "wait it out" and "build workarounds",
+  thereby reducing engineering bottlenecks.
+* Increase quality.
+* Increase employee happiness.
+* To increase success of new hires.
+* To build actionable documentation.
+
+### Provide clear InnerSource principles
+
+Once teams understand which problems InnerSource will help them address, the next
+step is to explain which principles help address these challenges. 
+
+Based on basic open source development principles the following guidelines 
+have been proven successful:
```

Comment:
```suggestion
Those involved with the InnerSource initiative in the organisation need to
understand two perspectives:

Those that should adopt InnerSource need to understand two perspectives:

### Why does the organisation want to adopt InnerSource?

In the past InnerSource has proven to be successful to solve several issues
commonly found in organisations. Instead of going for generalizations, try to
exactly identify the solutions that match the challenges of your organisation -
preferably with those affected by the change you want to see.

Some challenges that others have addressed by following InnerSource
best practices:

* Reduce development silos caused by excessive ownership culture.
* Increase innovation speed by reducing time spent solving similar issues by
  fostering healthy code reuse.
* Increase development speed by better cross team collaboration.
* Solve project/ team dependencies beyond "wait it out" and "build workarounds",
  thereby reducing engineering bottlenecks.
* Increase quality.
* Increase employee happiness.
* To increase success of new hires.
* To build actionable documentation.

### Provide clear InnerSource principles

Once teams understand which problems InnerSource will help them address, the next
step is to explain which principles help address these challenges.

Based on basic open source development principles the following guidelines
have been proven successful:
```

Fix linter issues